api/src/test/java/com/exelaration/abstractmemery/controllers/MemeControllerTest.java

```diff
@@ -65,4 +70,37 @@ public void MemeControllerTest_WhenNullMemePosts_ExpectStatus400() throws Except
                 .content(new ObjectMapper().writeValueAsString(testMeme)))
         .andExpect(badRequest);
   }
+
+  @Test
+  public void getMeme_WhenImageIsPresent_ExpectStatus200() throws Exception {
```

Comment:
What is the difference between this test and the one that starts on line 97

Reply:
Good catch. I need to adjust the last test